nanopub/java_wrapper.py

```diff
@@ -4,18 +4,28 @@
 from typing import Union
 
 import rdflib
+import requests
 
 from nanopub.definitions import PKG_FILEPATH
 
 # Location of nanopub tool (currently shipped along with the lib)
 NANOPUB_SCRIPT = str(PKG_FILEPATH / 'np')
+NANOPUB_TEST_SERVER = 'http://test-server.nanopubs.lod.labs.vu.nl/'
```

Comment:
Wondering if we should have a central config file that specifies ```NANOPUB_TEST_SERVER```, ```NANOPUB_JAVA_VERSION``` etc, just so we can see at a glance how the library is configured? Or do you think it's better keeping the settings local?

Reply:
Yeah I think that makes sense. I used to have that in most projects I worked on. I will do that in a separate PR.